docs/diagrams/AddApptCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+Actor user as user USER_COLOR
+
+box Logic LOGIC_COLOR_T1
+participant "LogicManager" as Logic LOGIC_COLOR
+participant "AddApptCommand" as AddApptCommand LOGIC_COLOR
+participant "CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "Model" as Model MODEL_COLOR
+participant "AddressBook" as AddressBook MODEL_COLOR
```

Comment:
I believe we would need to put `:` beside all the class name 

![image](https://github.com/user-attachments/assets/b087a9d1-edad-433b-9dfc-576c2128dbf5)

Reply:
Yes, unless we are calling static method, then we call from class itself. Else should have `:`
![image](https://github.com/user-attachments/assets/7b3649fc-7e1f-475a-a6a3-4241806a1ed8)